frontend/stores/notifications/useNotificationsManagementStore.ts

```diff
@@ -55,22 +55,24 @@ export const useNotificationsManagementStore = defineStore('notifications-manage
     ).then(() => {
       // using optimistic ui here to avoid calling the api after delete
       settings.value.paired_devices
-      = [ ...settings.value.paired_devices.filter(device => device.id !== id) ]
+        = [ ...settings.value.paired_devices.filter(device => device.id !== id) ]
     }).then(() => refreshOverview())
   }
   const setNotificationForPairedDevice = async ({
     id,
+    name,
     value,
   }: {
     id: number,
+    name?: string,
     value: boolean,
   }) => {
     await fetch<PutUserNotificationSettingsPairedDevicesResponse>(
       'NOTIFICATIONS_MANAGEMENT_PAIRED_DEVICES_SET_NOTIFICATION',
       {
         body: {
           is_notifications_enabled: value,
-          name: id,
+          ...(name && { name }),
```

Comment:
```suggestion
          name,
```
I think this should work too, as `ofetchs serializer` is smart enough to omit `name: undefined`

Reply:
Good to know 👌